save-demo-cpg/build.gradle.kts

```diff
@@ -20,6 +20,7 @@ repositories {
         }
     }
     mavenCentral()
+    mavenLocal()
```

Comment:
a temporary till `kotlin-tree-jna` is released to github packages

Reply:
added a link to fork

Reply:
need to remove mavenLocal when artifacts will be ready

Reply:
Removed with 666848983427d480af4983e71a28903b648e8e3d.